src/command_modules/azure-cli-storage/azure/cli/command_modules/storage/azcopy/util.py

```diff
@@ -0,0 +1,138 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+
+import os
+import json
+import platform
+import subprocess
+import datetime
+from six.moves.urllib.parse import urlparse
+from azure.cli.core._profile import Profile
+from knack.log import get_logger
+
+logger = get_logger(__name__)
+
+
+STORAGE_RESOURCE_ENDPOINT = "https://storage.azure.com"
```

Comment:
should we consider the end points in other clouds env, such as China cloud?

Reply:
I will confirm with service team to see if this endpoint works all the clouds.